### DISCUSSION TITLE
Football events on Attendize

### DISCUSSION THREAD

**Discussion by PrzemekSkw** [Attendize/Attendize#1138] [General]
Hello, I see Attendize is great ticketing platform but I wonder if it's any plugin that can make stadion view for football events. Where customers can choose places like on screenshoot?
![image](https://github.com/Attendize/Attendize/assets/25219988/10f17ea2-269a-4ee3-a308-28578a13a2ad)
regards.

**Comment by emergingdzns** [comment]
I'm looking for the same thing. I've got a client that wants to sell tickets to their custom events where there are different seating options and the buyer needs to be able to select their seats. I know it would be nearly impossible to have a one-size-fits-all solution but I wonder how hard it would be to inject a step in the checkout process where the buyer can select the seats based on the ticket type.
